etc/travis_test.sh

```diff
@@ -10,30 +10,27 @@ AnyFailures=No
 # check code formatting
 find . -name '*.c' -exec clang-format -i {} \;
 find . -name '*.h' -exec clang-format -i {} \;
-git diff --exit-code -- . # detect if there are any diffs
+git diff --exit-code -- '*.c' '*.h' # detect if there are any diffs
```

Comment:
Why this change? My best guess is: for the sake of `generated_path.gi`. But that is in `.gitignore`, so it shouldn't matter... so...?

Reply:
This is basically a leftover. When I tried to just do `Pkg.build()` with the changed travis `JULIA_PROJECT`, Nemo was added to the `Project.toml`. I can remove this again.